packages/mobile/audio-service.js

```diff
@@ -0,0 +1,9 @@
+import TrackPlayer from 'react-native-track-player'
+
+module.exports = async () => {
```

Comment:
Possible to make this ts?

Reply:
didn't know if it made sense to make this ts bc it's in the root dir with all of the other js config and setup files. No strong opinion on my end about this though